Charon/filetypes/OpenPackagingConvention.py

```diff
@@ -298,7 +298,11 @@ def _processAliases(self, virtual_path: str) -> str:
 
         # Replace all aliases.
         for regex, replacement in self._aliases.items():
-            virtual_path = re.sub(regex, replacement, virtual_path)
+            if regex.startswith("/"):
+                expression = r"^" + regex
```

Comment:
```suggestion
                expression = rf"^{regex}"
```
Double checked this. Order of `rf` doesn't seem to matter.

Reply:
On second thought; now that I have taken a closer look at the code, this is a nice fail-safe but isn't this 'just' a case of an incorrect alias definition?

If you look at libCharon/Charon/filetypes/UltimakerFormatPackage.py, you see that all aliases are neatly prefixed with "^/"...

Reply:
Indeed in UltimakerFormatPackage there is a `^/` fixed. There `expression = rf"^{regex}"` doesn't fit here...
On the other hand for replacement of paths, usually `/` refers to a top-level directory. It seems possible that when using it, even developers may forget putting `^` at the beginning (and consider the method as merely a regex replacement, not a path handler).

Therefore I'll just keep

```Python
            if regex.startswith("/"):
                expression = r"^" + regex
```

Reply:
My first suggestion still stands ;)